File: sqlConnect.php
<?php
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

//$sql = "SELECT name, age, gradeLevel FROM students ORDER BY name";
//$sql = "SELECT name, age, gradeLevel FROM students WHERE name='Deric' AND age=17 LIMIT 1";
//$sql = "DELETE FROM students WHERE id = 10"

$sql = "SELECT * FROM students"
$result = $conn->query($sql);

if ($result->num_rows > 0) {
     // output data of each row
     while($row = $result->fetch_assoc()) {
        echo $row['name']." is " . $row['age']." years old and in grade ".$row['gradeLevel'];
         echo "<br><br>"; 
     }
 } else {
    echo "0 results";
 }

$conn->close();
?>

File: sqlConnect.py
#import MySQLdb
import mysql.connector

# Open database connection
db = mysql.connector.connect(host="localhost",user="user",passwd="password",db="school" )

# prepare a cursor object using cursor() method.
cursor = db.cursor()

cur = db.cursor(dictionary=True)

# cursor = db.cursor(mysql.connector.cursors.DictCursor)

# Create a table as per requirement
sql = "SELECT * from students"

# execute SQL query using execute() method.
cur.execute(sql)

#fetch all data and print 
rows = cur.fetchall()
for row in rows:
	print(row)

# disconnect from server
db.close()
